src/components/KymaCompanion/components/KymaCompanion.tsx

```diff
@@ -1,18 +1,52 @@
 import { useTranslation } from 'react-i18next';
 import { Button, Card, Title } from '@ui5/webcomponents-react';
-import { useRecoilState } from 'recoil';
+import { useRecoilState, useRecoilValue, useSetRecoilState } from 'recoil';
 import {
   ShowKymaCompanion,
   showKymaCompanionState,
 } from 'components/KymaCompanion/state/showKymaCompanionAtom';
 import Chat from './Chat/Chat';
+import { useEffect, useState } from 'react';
+import { sessionIDState } from '../state/sessionIDAtom';
+import { clusterState } from 'state/clusterAtom';
+import getPromptSuggestions from '../api/getPromptSuggestions';
 import './KymaCompanion.scss';
 
 export default function KymaCompanion() {
   const { t } = useTranslation();
   const [showCompanion, setShowCompanion] = useRecoilState<ShowKymaCompanion>(
     showKymaCompanionState,
   );
+  const [suggestions, setSuggestions] = useState<any[]>([]);
```

Comment:
shouldn't be like this?
```suggestion
  const [suggestions, setSuggestions] = useState<string[]>([]);
```

Reply:
👍